src/NuGet.Core/NuGet.Commands/PushRunner.cs

```diff
@@ -30,9 +30,18 @@ public static class PushRunner
 
             PackageUpdateResource packageUpdateResource = await CommandRunnerUtility.GetPackageUpdateResource(sourceProvider, source);
 
-            string symbolsSource = !noSymbols
-                ? NuGetConstants.DefaultSymbolServerUrl
-                : string.Empty;
+            // only push to SymbolSource when the actual package is being pushed to the official NuGet.org
+            string symbolsSource = string.Empty;
+
+            Uri sourceUri;
+            if (!noSymbols && Uri.TryCreate(source, UriKind.RelativeOrAbsolute, out sourceUri))
+            {
+                if (sourceUri.Host.Equals(NuGetConstants.NuGetHostName, StringComparison.OrdinalIgnoreCase) // e.g. nuget.org
+                    || sourceUri.Host.EndsWith("." + NuGetConstants.NuGetHostName, StringComparison.OrdinalIgnoreCase)) // *.nuget.org, e.g. www.nuget.org
```

Comment:
Do you need to check the scheme for http? Like, what if the path is a local share \\foo.nuget.org\bar